x/evm/keeper/keeper.go

```diff
@@ -20,36 +20,37 @@ import (
 // Keeper wraps the CommitStateDB, allowing us to pass in SDK context while adhering
 // to the StateDB interface
 type Keeper struct {
-	csdb     *types.CommitStateDB
-	cdc      *codec.Codec
-	blockKey sdk.StoreKey
-	txCount  *count
-	bloom    *big.Int
+	cdc           *codec.Codec
+	storeKey      sdk.StoreKey
+	CommitStateDB *types.CommitStateDB
+	TxCount       *count
+	Bloom         *big.Int
 }
 
+// TODO: move to types
```

Comment:
Intended to do this in this PR ? To give context on this, just a counter to be used to prepare statedb for log generation in EVM

Reply:
I will do a cleanup later next week. It is out of scope for this PR